appveyor.yml

```diff
@@ -32,12 +36,24 @@ build_script:
     $manifest = "src\Docker.PowerShell\Docker.psd1"
     (Get-Content $manifest -Raw) -replace "ModuleVersion.+","ModuleVersion = '$psversion'" | Out-File $manifest
 - ps: dotnet restore
-- ps: dotnet publish -r win -o bin -c Release src/Docker.PowerShell
-- ps: New-ExternalHelp -Path src/Docker.PowerShell/Help -OutputPath bin
+- ps: dotnet publish -r win -o $pwd\bin -c Release $pwd\src\Docker.PowerShell
+- ps: New-ExternalHelp -Path src\Docker.PowerShell\Help -OutputPath bin
```

Comment:
I actually don't... I only added it to dotnet publish to make sure we avoid the publish bug that hits when using relative paths.  But if you prefer consistency here, I can change the others.